recipe/meta.yaml

```diff
@@ -57,3 +57,6 @@ about:
 extra:
   recipe-maintainers:
     - hajapy
+    - rluria14
+    - oblute
+    - ndmaxar
```

Comment:
@oblute and @ndmaxar, are you aware you are being asked to be maintainers on this feedstock?

Reply:
@hajapy yes, happy and willing to co-maintain 👍

Reply:
Yep!